assets/css/3rd-party/themes/course/quiz.scss

```diff
@@ -1,21 +1,41 @@
-.sensei-course-theme__quiz {
+.wp-block-sensei-lms-quiz {
+	.wp-block-sensei-lms-question-answers {
 
-	&__main-content {
-		.wp-block-post-title {
-			margin-bottom: 3.75rem;
+		input[type=file]::file-selector-button,
```

Comment:
Same question as https://github.com/Automattic/sensei/pull/7124/files#r1309109489.

Reply:
Updated it here https://github.com/Automattic/sensei/pull/7124/commits/bcfe828c2da7ad6d0c9920b3d5f4538e6c451b99